mne_bids/tests/test_write.py

```diff
@@ -273,6 +273,37 @@ def test_fif(_bids_validate):
     data = _from_tsv(participants_tsv)
     assert data['age'][data['participant_id'].index('sub-01')] == '9'
 
+    # check to make sure participant data is overwritten, but keeps the fields
+    data = _from_tsv(participants_tsv)
+    participant_ind = data['participant_id'].index(f'sub-{subject_id}')
```

Comment:
```suggestion
    participant_idx = data['participant_id'].index(f'sub-{subject_id}')
```

I think we tend to use "idx" instead of "ind", maybe look around in the code to see what is more common